cylc/flow/tui/data.py

```diff
@@ -33,7 +33,7 @@
           meanElapsedTime
         }
       }
-      familyProxies(states: $taskStates) {
+      familyProxies(exids: ["root"], states: $taskStates) {
```

Comment:
Eliminates `if familyProxy == root`. Only place where root is used—I think— is when computing cycle points. It's also used when it's the first parent, but that data is still in the response.